hieunc-guestbook-dojo/static/js/guestbook/store/_GreetingStore.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * Created by hieunc on 27/11/2015.
+ */
+
+define([
+	"dojo/_base/declare",
+	"dojo/_base/lang",
+	"dojo/Deferred",
+	"dojo/request/xhr"
+], function (declare, lang, Deferred, xhr) {
+	return declare("_GreetingStore", null, {
```

Comment:
- đặt tên class theo cấu trúc như sau: `path.to.file.ClassName`
- tên class trùng với tên file (trừ trường hợp là SubClass)
- hãy bỏ dấu `_`, vì theo convention, `_` dùng để chỉ những class/method/variable là private
- check lại trên các class/method/variable khác